Expecto.netcore/Expecto.netcore.fsproj

```diff
@@ -1,54 +1,33 @@
-<Project ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
-  <Import Project="$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props" />
+<Project Sdk="FSharp.NET.Sdk;Microsoft.NET.Sdk">
+
   <PropertyGroup>
     <AssemblyName>Expecto</AssemblyName>
     <Version>1.1.2</Version>
     <OutputType>Library</OutputType>
     <TargetFramework>netstandard1.6</TargetFramework>
-    <DebugType>pdbonly</DebugType>
-  </PropertyGroup>
-  <PropertyGroup Condition=" '$(Configuration)' == 'Release' ">
-    <DefineConstants>$(DefineConstants);RELEASE</DefineConstants>
+    <DebugType>portable</DebugType>
+    <DefineConstants Condition=" '$(TargetFramework)' == 'netstandard1.6' ">
```

Comment:
just a style note (i learned that recently and i like that), using multiline help with changes later (less diff) and to read list if long.

```xml
<DefineConstants Condition=" '$(TargetFramework)' == 'netstandard1.6' ">
    NETSTANDARD1_6;
    RESHAPED_REFLECTION;
    @(DefineConstants)
</DefineConstants>
```